errors/mm/custom-lobby-config-invalid.md

```diff
@@ -8,3 +8,34 @@ http_status = 400
 # Matchmaker Custom Lobby Config Invalid
 
 The given custom lobby config is invalid. This is most likely because it exceeds the maximum size limit of 16KiB.
+
+## Publicity errors
+
+You may encounter an error such as `"public" publicity not allowed with this custom game mode`. This occurs when the publicity of the `/create` request is not allowed by the custom game mode config configured for this game.
+
+Given this version config:
+
+```yaml
+matchmaker:
+  game_modes:
+    default:
+      actions:
+        create:
+          enabled: true
+          enable_public: false
+          enable_private: true
+```
+
+The following request will not work:
+
+```bash
+curl \
+	-X POST \
+	-H "Content-Type: application/json" \
+	-d '{ "game_mode": "default", "publicity": "public" }' \
+	'https://api.rivet.gg/matchmaker/lobbies/create'
+```
```

Comment:
curl is _scary_, thoughts on using ts instead? imo it's a universally understandable language.

Reply:
needs ts code